app/assets/stylesheets/wcag.scss

```diff
@@ -291,4 +291,9 @@ html {
       color: var(--secondary);
     }
   }
+
+  .cooked mark,
+  .d-editor-preview mark {
+    background-color: yellow; // resets to browser default
```

Comment:
Does `unset` work here? This assumes `yellow` is the default which might not always be the case.

Reply:
No, it doesn't, it removes the background (instead resetting to browser default).

Reply:
How about `revert`? 😃

Reply:
Yes `revert` works, but doesn't have great browser support. Will stick with yellow.